egasub/submission/init.py

```diff
@@ -11,6 +11,12 @@ def init_workspace(ctx,ega_submitter_account=None,ega_submitter_password=None,ic
     ctx.obj['LOGGER'].info('Note: information collected below will be stored in')
     ctx.obj['LOGGER'].info('      \'.egasub/config.yaml\' which can be edited later.')
     
+    projects = ["BLCA-CN","BOCA-FR","BOCA-UK","BRCA-EU","BRCA-FR","BRCA-KR","BRCA-UK","BTCA-JP","BTCA-SG","CLLE-ES","CMDI-UK",
+            "COCA-CN","EOPC-DE","ESAD-UK","ESCA-CN","GACA-CN","LAML-CN","LAML-KR",
+            "LIAD-FR","LICA-CN","LICA-FR","LIHM-FR","LINC-JP","LIRI-JP","LUSC-CN","LUSC-KR","MALY-DE","MELA-AU","ORCA-IN","OV-AU",
+            "PACA-AU","PACA-CA","PAEN-AU","PAEN-IT","PBCA-DE","PRAD-CA","PRAD-UK","RECA-CN","RECA-EU","SKCA-BR","THCA-SA",
+        ]
```

Comment:
should avoid hard-code the list in the code, but let's start with this for now as project list is relatively stable.